packages/insomnia-app/app/models/space.ts

```diff
@@ -14,13 +14,15 @@ export const BASE_SPACE_ID = 'base-space';
 
 interface BaseSpace {
   name: string;
+  remoteId?: string;
 }
 
 export type Space = BaseModel & BaseSpace;
 
 export function init(): BaseSpace {
   return {
     name: 'My Space',
+    remoteId: undefined, // this is necessary for the model init logic to work properly
```

Comment:
```suggestion
  remoteId: string | undefined;
}

export type Space = BaseModel & BaseSpace;

export function init(): BaseSpace {
  return {
    name: 'My Space',
    remoteId: undefined, // this is necessary for the model init logic to work properly
```

The next version of TypeScript (4.4) will have (https://github.com/microsoft/TypeScript/pull/43947) some logic that distinguishes between these two things.  Let's try to stay accurate about it now so it'll be easier to transition when the time comes (soon).

Reply:
I think we should consider using `null` for this since it is more specific and will serialize when we import/export:

https://github.com/dimitropoulos/insomnia/commit/ede3ca97d82e4254d2599edb7a428099cbe68fd6

Reply:
Nice catch 👍🏽